src/Arcus.Messaging.Abstractions/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -64,6 +64,8 @@ public static MessageHandlerCollection AddMessageRouting(this IServiceCollection
             Guard.NotNull(services, nameof(services), "Requires a set of services to add the message routing");
             Guard.NotNull(implementationFactory, nameof(implementationFactory), "Requires a function to create the message router");
 
+            services.AddApplicationInsightsTelemetryWorkerService();
```

Comment:
Just wondering why we have the `WorkerService` part in the methodname ?

Reply:
This is an extension from Microsoft. This will make sure that the `TelemetryClient` is registered and ready to be used when built-in Microsoft dependencies are being tracked in custom message handlers.